lib/ItemRow/index.js

```diff
@@ -20,6 +28,12 @@ const ItemRow = ({ children, label, indicator, participants, stacked }) => {
       {participants && (
         <span className="item-row__participants">{participants}</span>
       )}
+      {commentCount !== '0' && (
```

Comment:
```suggestion
      {commentCount >= '0' && (
```

That way we cover ourselves if the comment count comes back as `-1` for whatever weird reason.

Reply:
Sadly its a string rather than an int :(

Reply:
Good point! That won't work then XD